tests/Fixer/FunctionNotation/FunctionCompactNullableTypehintFixerTest.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\Fixer\FunctionNotation;
+
+use PhpCsFixer\Tests\Test\AbstractFixerTestCase;
+
+/**
+ * @requires PHP 7.1
+ *
+ * @author Jack Cherng <jfcherng@gmail.com>
+ *
+ * @internal
+ *
+ * @covers \PhpCsFixer\Fixer\FunctionNotation\FunctionCompactNullableTypehintFixer
+ */
+final class FunctionCompactNullableTypehintFixerTest extends AbstractFixerTestCase
+{
+    /**
+     * @param string      $expected
+     * @param null|string $input
+     *
+     * @dataProvider providePhp71FixCases
+     */
+    public function testFix($expected, $input = null)
+    {
+        $this->doTest($expected, $input);
+    }
+
+    public function providePhp71FixCases()
+    {
+        return [
+            [
+                '<?php function foo(?int $param): ?int {}',
+            ],
+            [
+                '<?php function foo(/**? int*/$param): ?int {}',
+                '<?php function foo(/**? int*/$param): ? int {}',
```

Comment:
I do know it's sad, but please add test containing
- `? /* foo */ int` 
- `? /** foo */ int` 
-
```php
? // foo
int
```

Reply:
Nice catch.